Neos.Flow/Classes/Mvc/Controller/AbstractController.php

```diff
@@ -234,12 +243,8 @@ protected function forward($actionName, $controllerName = null, $packageKey = nu
      */
     protected function forwardToRequest(ActionRequest $request)
     {
-        $packageKey = $request->getControllerPackageKey();
-        $subpackageKey = $request->getControllerSubpackageKey();
-        if ($subpackageKey !== null) {
-            $packageKey .= '\\' . $subpackageKey;
-        }
-        $this->forward($request->getControllerActionName(), $request->getControllerName(), $packageKey, $request->getArguments());
+        $nextRequest = clone $request;
```

Comment:
we only need to clone as far as i know, because that will unset the dispatch state? But that will be hopefully be fixed via https://github.com/neos/flow-development-collection/pull/3301